### PR TITLE
Clean-up and improve Isentropic Analysis notebook

### PR DESCRIPTION
**Pull request by dopplershift**
Removes a few needless cells near the end that look at units. Also rework to use metpy accessor and simplify the process of selecting common levels in the fields.

### CODE REVIEW

File: notebooks/MetPy_Advanced/Isentropic Analysis.ipynb [hunk 1]
```diff
@@ -239,27 +240,39 @@
...
-    "# Due to a different number of vertical levels find where they are common\n",
-    "lev_hght = temperature.coords['isobaric4'] * units.Pa\n",
-    "lev_uwnd = u.coords['isobaric'] * units.Pa\n",
-    "_, _, common_ind = np.intersect1d(lev_uwnd, lev_hght, return_indices=True)\n",
-    "temperature = temperature[common_ind, :, :]\n",
-    "# Reassign units to variables\n",
-    "temperature.attrs['units'] = 'kelvin'\n",
-    "lev_uwnd.attrs['units'] = 'Pa'"
+    "# Get common pressure levels as a data array\n",
+    "press = press.metpy.sel(vertical=common_levels)"
```
Comment (zbruick) [inline notebooks/MetPy_Advanced/Isentropic Analysis.ipynb]: Should we be going back and reworking other examples that use this common levels hack with this method that you have here?

Comment (dopplershift) [reply notebooks/MetPy_Advanced/Isentropic Analysis.ipynb]: Maybe? Probably on a case-by-base basis. Anywhere we're doing something more complicated to handle differing vertical levels.

**Comment by dopplershift** [comment]
Seems to be a weird bug in MetPy, but only in master. See Unidata/MetPy#1124. In the meantime, I've adjusted the code to (I think) work on 0.10.2 and master.

**Comment by zbruick** [comment]
Restarted Travis. Errored out on the NCSS notebook with THREDDS issues. Otherwise, this looks good.

**Pull request merged**